rust/agama-migrate-wicked/src/interface.rs

```diff
@@ -68,13 +72,97 @@ pub struct Address {
     pub local: String,
 }
 
-impl From<Interface> for model::Connection {
-    fn from(val: Interface) -> Self {
-        let mut con = model::Connection::new(val.name.clone(), DeviceType::Ethernet);
-        let base_connection = con.base_mut();
-        base_connection.interface = val.name.clone();
-        base_connection.ip_config = (&val).into();
-        con
+#[derive(Debug, PartialEq, Default, Serialize, Deserialize)]
+pub struct Bond {
+    pub mode: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub miimon: Option<Miimon>,
+    #[serde(deserialize_with = "unwrap_slaves")]
+    pub slaves: Vec<Slave>,
```

Comment:
```suggestion
    pub ports: Vec<Slave>,
```

Reply:
I don't understand this suggestion, `ports` doesn't exist in wickeds xml, right?
Btw, Marius suggested to not use `<slaves>` in the master because it will get reworked in the future, right? And I think this maps more nicely to NM anyway. Though where does the `primary` info get stored in the future?

Reply:
It's because of advises like here https://github.com/yast/yast-network/pull/1277 

Maybe we should do it only in models.rs but not in interface.rs?

Reply:
Of course, it does not make sense to modify it everywhere as also commented in the PR:

> What was not replaced:
> 
> some occurrences which more / less define an API to other components in the system (sysconfig, NetworkManager). We could not replace these as respective components still use these terms. Even some low level pieces of our code can still use old naming to avoid confusion. This should be replaced when the API becomes obsolete.

But we should use inclusive naming when it is possible.